### ISSUE TITLE
Repeater pagination messes up URL query parameters

### ISSUE THREAD

**Issue by zuroma999** [soflyy/oxygen-bugs-and-features#2512]
**Describe the bug**
If there are query args in the URL of a page with a repeater, the pagination links alter the last query argument in the URL, as well as make a duplicate of all the query arguments. This messes up any custom code relying on the query arguments.

**A link to a Sandbox site where the bug has been reproduced https://oxygenbuilder.com/try? (REQUIRED)**
As an example on this page:
https://gorgeous-parrot.w5.wpsandbox.pro/test?arg1=val1&arg2=val2&arg3=val3

if you paginate you will see the last URL query argument get messed up, and also all the arguments get duplicated.

**Comment by supermixture** [comment]
Yes, I have the same problem and hopefully it will be fixed soon. I am in the middle of setting up WooCommerce's categories and tags and it's not fun. Kevin has/is being helpful, thanks to him.

**Comment by maltmann-muc** [comment]
Have the same issue. Did some investigation with WordPress 6.1, Oxygen 4.1.1.

Assume archive URL  /blog/?A=1&B=2
is changed to pagination URL /blog/page/2/?A=1&B=2#038;B=2

Issue found in /plugins/oxygen/component-framework/components/classes/dynamic-list.class.php, line 897:

`'base' => str_replace( $big, '%#%', esc_url( get_pagenum_link( $big ) ) ),`

should be 

`'base' => str_replace( $big, '%#%', get_pagenum_link( $big, false ) ),`
